### PR TITLE
Rework of some packer shell scripts

### PR DESCRIPTION
**Pull request by mcrmonkey**
~ Lumped all op user bits in to one useradd command and have it create
the user homedir and set the shell
~ Used shell expansion to create additional dirs in user home
~ Changed ordering in the script so heavy apt work is done earlier for
common stuff. This would allow other things to be sectioned out

- Removed `.DS_store` and added to gitignore
- Removed a random whitespace on the end of the `do_key` line

### CODE REVIEW

**Comment by pry0cc** [comment]
Looks really great dude!

Can’t wait to test it!

**Comment by pry0cc** [comment]
Just span up an instance with this build, I can't seem to login over SSH?

Have you tested this?

**Comment by pry0cc** [comment]
Ok I did some more testing... It was the `"mkdir -p /home/op/{.ssh,c2,lists}",` line, I guess that syntax shorthand only works in bash itself?

Anyway - I moved it to use three different mkdir commands. What was happening was that the SSH authorized keys wasn't getting moved because the `.ssh` mkdir was failing...

I'm going to close this, and merge with my changes, I'm sorry I can't get you in the merge tree, but I'll add you to contributors! I don't know how to git that well :P

**Comment by mcrmonkey** [comment]
ahh bother...
packer runs `sh` as the shell not bash so the param expansion doesn't work here

File: packer.json [hunk 1]
```diff
@@ -37,25 +38,19 @@
...
+                "pip install aiodnsbrute docker-py",
                 "gem install bundler",
                 "systemctl start docker",
                 "systemctl enable tor",
                 "ufw --force enable",
-                "useradd -G sudo,docker op",
-                "mkdir -p /home/op/.ssh/",
-                "chsh -s /usr/bin/zsh op",
+                "useradd -G sudo,docker -s /usr/bin/zsh -m op",
+                "mkdir -p /home/op/{.ssh,c2,lists}",
```
Comment (mcrmonkey) [inline packer.json:52]: This could be spread out as this for single line
```
mkdir -p /home/op/.ssh /home/op/c2 /home/op/lists
```

**Pull request closed without merging**